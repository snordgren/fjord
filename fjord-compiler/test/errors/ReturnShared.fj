module errors.ReturnShared

returnShared : Int -> Int
returnShared x =
  x
