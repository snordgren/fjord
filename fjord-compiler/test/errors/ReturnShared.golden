errors/ReturnShared.fj:5:3:
  |
5 |   x
  |   ^^
expected unique value
